ci/build_libcudacxx.sh

```diff
@@ -2,14 +2,21 @@
 
 source "$(dirname "$0")/build_common.sh"
 
+# lit only knows ccache, but we want sccache for the cloud storage, so alias it
+if ! [ -x "$(command -v ccache)" ]; then
+    sudo ln -s /usr/bin/sccache /usr/bin/ccache
+fi
+
 cmake -S .. -B ../build \
     -DCCCL_ENABLE_THRUST=OFF \
     -DCCCL_ENABLE_LIBCUDACXX=ON \
     -DCCCL_ENABLE_CUB=OFF \
     -DCCCL_ENABLE_TESTING=OFF \
     -DCMAKE_BUILD_TYPE=Release \
     -DCMAKE_CXX_COMPILER=${HOST_COMPILER} \
+    -DCMAKE_CXX_COMPILER_LAUNCHER=sccache \
     -DCMAKE_CUDA_ARCHITECTURES=${GPU_ARCHS} \
+    -DCMAKE_CUDA_COMPILER_LAUNCHER=sccache \
```

Comment:
This isn't necessary as it is already being set in the environment variables.

Reply:
Removed